prime-router/src/main/kotlin/azure/DatabaseLookupTableAccess.kt

```diff
@@ -151,21 +152,28 @@ class DatabaseLookupTableAccess(private val db: DatabaseAccess = DatabaseAccess(
     }
 
     /**
-     * Create a new table [version] for a [tableName] using the provided [tableData].
+     * Create a new table [version] for a [tableName] using the provided [tableData].  The force flag is use
```

Comment:
Change the text here of force to [force], so it shows up correctly as an argument in kdoc

Reply:
Done.